RELEASE-NOTES.md

```diff
@@ -1,3 +1,33 @@
+### 0.11.0
+- Added virtual machine with automating configuration of golem and concent for testing purpose (#146, #147, #148, #182, #186).
+- Added liveness and readiness probe to geth (#44).
+- Added support for enabling routing based on golem-messages version on a single cluster (#289).
+- Added ability to manage resource limits for applications on different clusters (#257).
+- Added ability to switching between an internal and external geth instance (#285).
+- Added ability to generate a key pair for signing-service (#335).
```

Comment:
`ability to generate` -> `helper script and instructions for generating`
`signing-service` -> `Signing Service`